seed/static/seed/tests/protractor-tests/protractorConfig.js

```diff
@@ -33,20 +23,6 @@ exports.config = {
     }
   },
   capabilities: {
-    browserName: 'chrome',
-    chromeOptions: {
-      args: [
-        '--headless',
-        '--disable-gpu',
-        '--no-sandbox',
-        '--window-size=800,600',
-        '--disable-extensions',
-        '--disable-dev-shm-usage'
-      ]
-    }
```

Comment:
We don't actually run our CI tests headless. On travis we ran protractorConfigCoverage.js, which is essentially this exact file but not headless. I deleted that file and updated this one to consolidate things.